lib/server/bootevent.js

```diff
@@ -137,6 +137,10 @@ function boot (env, language) {
     if (env.settings.isEnabled('alexa')) {
       ctx.alexa = require('../plugins/alexa')(env, ctx);
     }
+    
+    if (env.settings.isEnabled('googlehome')) {
+      ctx.googleHome = require('../plugins/googlehome')(env, ctx);
+    }
 
     next( );
   }
```

Comment:
@PieterGit let's try to figure out if there's a way to reduce plugin-specific code in this section